run/filesystem/test/system.test.js

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const assert = require('chai').expect;
+const chai = require('chai');
+const {GoogleAuth} = require('google-auth-library');
+const {execSync} = require('child_process');
+const chaiHttp = require('chai-http');
+chai.use(chaiHttp);
+const auth = new GoogleAuth();
+
+const getEnvironmentVariable = (variable, defaultValue) => {
```

Comment:
This file reads very different from our other sample tests https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/run/system-package/test/system.test.js which may show more idiomatic JS. Tagging @grayside for Node.js SME

Reply:
This function is pretty much [these lines of code](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/547f45f7320fbd65de48967407d3b7c066c4efc3/run/system-package/test/system.test.js#L33C1-L42C4)

Reply:
Refactored to align with existing samples